nav2_costmap_2d/test/obstacle_tests.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
Should this file be here? It's a ROS style launch file, I think it would need to be replaced for ROS2 with a python launch file.

Reply:
So these have been floating around in this repo as of today. I agree that they would not work, but I dont think we should remove them unless we either make a ticket to bring back this same functionality or remove the tests outright.

I think given the scope of this PR already is large, I would just leave it and make a separate ticket after this is merged to enable unit testing of this in ROS2

Reply:
@bpwilcox - does #121 mean to add these tests? It's not clear to me please verify what it means exactly